package.json

```diff
@@ -34,7 +34,7 @@
   "homepage": "https://github.com/cozy/cozy-contacts#readme",
   "devDependencies": {
     "babel-preset-cozy-app": "1.3.0",
-    "cozy-scripts": "1.12.1",
+    "cozy-scripts": "1.11.0",
```

Comment:
Why are we downgrading cozy-scripts in this PR?

Reply:
See the chan. renovate has updated CS during the night and we has an issue with this release. 

We expect a new release today / tomorrow.

Reply:
Mmh alright, but renovate is going to autoupdate it again, and if the newest version isn't out by then we'll have the same problem.